patches/chromium/add_electron_deps_to_license_credits_file.patch

```diff
@@ -0,0 +1,51 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Charles Kerr <charles@charleskerr.com>
+Date: Tue, 9 Aug 2022 12:35:36 -0500
+Subject: add electron deps to license credits file
+
+Ensure that licenses for the dependencies introduced by Electron
+are included in `LICENSES.chromium.html`
+
+diff --git a/tools/licenses.py b/tools/licenses.py
+index a58dbf44370baabbfa2986c734c96a210cc16f1d..af31bdd33a83fb10d748e47f3c4e4a1fafea7e09 100755
+--- a/tools/licenses.py
++++ b/tools/licenses.py
+@@ -347,6 +347,38 @@ SPECIAL_CASES = {
+         "License File":
+         "/third_party/swiftshader/third_party/SPIRV-Headers/LICENSE",
+     },
++
++    os.path.join('third_party', 'electron_node'): {
++        "Name": "Node.js",
++        "URL": "https://github.com/nodejs/node",
++        "License": "MIT",
++        "License File": "/third_party/electron_node/LICENSE",
++    },
++    os.path.join('third_party', 'nan'): {
```

Comment:
We don't distribute nan, so it doesn't need to be in the license file.

Reply:
removed in 5aee785f24a745783d33de0753e747314f9837fc